changelog.d/9786.misc

```diff
@@ -0,0 +1 @@
+Apply `pyupgrade` across the codebase.
```

Comment:
The changelog is this line because I'll use it across other PRs that do `pyupgrade`, which'll then be merged into one line by towncrier.